cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config

```diff
@@ -36,7 +36,10 @@ write_files:
           WantedBy=cloud-init.service
 runcmd:
   - sudo systemctl daemon-reload
-  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics
+  - sudo dnf install -y http://ccrma.stanford.edu/planetccrma/mirror/fedora/linux/planetccrma/16/i386/planetccrma-repo-1.1-3.fc16.ccrma.noarch.rpm
+  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics kernel-rt tuned-profiles-realtime
+  - sudo grubby --set-default=$(ls /boot/vmlinuz-*rt*)
+  - sudo echo -e "isolated_cores=0,1\nisolate_managed_irq=Y" >/etc/tuned/realtime-variables.conf
```

Comment:
I think its possible to `sysctl` to achieve that, isnt it?

Reply:
> Hi Jordi,
> I am not sure I fully understand how is it going to work,
> the default kernel that will be used is the regular one (time-sliced?), and from the test suite we are going to boot the VM with the real-time kernel?

The default kernel will be the real time kernel. What I need to test (in progress) is to validate that using the real time kernel won't impact the usage of this container disk in the functional tests. It's the alternative to having to create a new container disk just for real time tests and having to manage it.

Reply:
> I think its possible to use `sysctl` to achieve that, isnt it?

I don't know, I need to check :smile:

Reply:
@ormergi it's not currently possible to change the settings with `sysctl`. As I understood from the feedback I've received, it's a work in progress.  The only other way is to inject these values as part of the kernel boot parameters.